earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -38,9 +43,55 @@ class StorageManaged {
     StorageManaged() : timestamp(0), filesize(0) {}
 };
 
+template<class AssetType, class AssetStorageType> class AssetHandleInterface;
+
+template<class AssetType, class AssetStorageType>
+class AssetSerializerLocalXML
+{
+  public:
+    AssetSerializerLocalXML() {}
+    std::shared_ptr<AssetType> Load(const std::string &boundref)
```

Comment:
Here and other places you can use AssetPointerType instead of std::shared_ptr. That makes it easier to switch between them.

Reply:
Changed as suggested.